third_party/docfx/templates/devsite/partials/uref/namespace.tmpl.partial

```diff
@@ -21,7 +21,11 @@
   {{^isEmbedded}}
     {{#children}}
       <h2><xref uid="{{uid}}" altProperty="fullName" displayProperty="name"/></h2>
+      {{#summary}}
+      {% verbatim %}

```

Comment:
We're playing Whack-a-Mole with `verbatim`. What if we mark everything inside the `body` tags as `verbatim` and remove the others?

Reply:
Looks like marking stuff verbatim has little to no side effect besides whitespaces we might be adding (if placed incorrectly) and more content being rendered (which should probably show?), so I think it sounds like a better idea.